proxysql-scheduler/local/build-binary-proxysql-scheduler

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+
+# ------------------------------------------------------------------------------
+# Declare all input variables
+# ------------------------------------------------------------------------------
+
+
+# ------------------------------------------------------------------------------
+# set working dir
+# ------------------------------------------------------------------------------
+#mkdir -p ${1:-./build}
+WORKDIR=$(cd ${1:-./build}; pwd -P)
+INSTALL_DIR=${WORKDIR}/DESTDIR
+SOURCEDIR=$(cd ${2:-$(dirname $0)/../sources/proxysql-scheduler}; pwd -P)
+
+pushd /tmp
+    #sudo wget -q https://golang.org/dl/go1.15.8.linux-amd64.tar.gz
+    #sudo tar -C /usr/local -xzf go1.15.8.linux-amd64.tar.gz
+    #export PATH=$PATH:/usr/local/go/bin
+    #sudo /usr/local/go/bin/go version
+    #sudo rm go1.15.8.linux-amd64.tar.gz
+    sudo apt update
+    sudo apt install -y golang
```

Comment:
this will work only on deb-based systems but on rpm based it will cause an error.

Reply:
The current goal is to have it working only on one platform. Particularly Ubuntu Focal (which is hardcodded when we create Docker container). We test here platform-independent solution, so no point at this moment to have a matrix job like PS/PXC.
If we need this in the future, I will have to rework it of course, but for now, this is exactly what we need.